v1.0/benchmarks/redis_callback_class.py

```diff
@@ -0,0 +1,56 @@
+from factor_graph import RedisCallbackClass
+from state import RedisNodeStateStore
+from node_update_functions import ALGORITHM_TO_UPDATE_FUNCTIONS
+from redis import Redis
+import timeit
+
+def timer(function_to_time):
+    def function_timer(*args, **kwargs):
+        setup()
+        t = timeit.Timer(function_to_time)
+        print(function_to_time.__name__, " benchmark: ", t.timeit(10))
+        takedown()
+    return function_timer
+
+def setup():
+    redis.flushall()
+
+    node_cache = RedisNodeStateStore()
+    node_cache.create_node_state(node_id, initial_messages, node_type,
+            node_data, stop_countdown)
```

Comment:
not sure where the args are supplied from

Reply:
these args are currently just specified in the ``if __name__ == '__main__':`` method